superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm/ValidatedInputField.tsx

```diff
@@ -26,14 +26,14 @@ const FIELD_TEXT_MAP = {
     helpText: t(
       'Copy the account name of that database you are trying to connect to.',
     ),
-    placeholder: 'e.g. world_population',
+    placeholder: t('e.g. world_population'),
   },
   warehouse: {
-    placeholder: 'e.g. compute_wh',
+    placeholder: t('e.g. world_population'),
```

Comment:
This is the only string you've actually changed here... just making sure this is intended, and the reference to "warehouse" isn't helpful to the UX.